### PR TITLE
FMWK-259 Add converters for commonly used Java types

### PR DESCRIPTION
**Pull request by agrgr**

### CODE REVIEW

File: src/main/java/org/springframework/data/aerospike/convert/AerospikeConverters.java [hunk 1]
```diff
@@ -109,4 +129,112 @@ public String convert(Enum<?> source) {
...
+
+    @ReadingConverter
+    enum StringToURLConverter implements Converter<String, URL> {
+        INSTANCE;
+
+        private static final TypeDescriptor source = TypeDescriptor.valueOf(String.class);
+        private static final TypeDescriptor target = TypeDescriptor.valueOf(URL.class);
+
+        public URL convert(String source) {
+
```
Comment (reugn) [inline src/main/java/org/springframework/data/aerospike/convert/AerospikeConverters.java:150]: remove empty line

**Review by reugn — COMMENTED** [review]
Make sure we have tests for all newly added converters.

File: src/main/java/org/springframework/data/aerospike/convert/AerospikeConverters.java [hunk 2]
```diff
@@ -47,6 +56,17 @@ static Collection<Object> getConvertersToRegister() {
         converters.add(LongToBooleanConverter.INSTANCE);
         converters.add(EnumToStringConverter.INSTANCE);
```
Comment (roimenashe) [inline src/main/java/org/springframework/data/aerospike/convert/AerospikeConverters.java:58]: Why separate the new ones from existing ones? can we remove this new empty line?

File: src/test/java/org/springframework/data/aerospike/convert/MappingAerospikeConverterTypesTests.java [hunk 3]
```diff
@@ -356,6 +362,107 @@ void ObjectWithByteArrayField() {
             new Bin("array", new byte[]{1, 0, 0, 1, 1, 1, 0, 0}));
     }
 
+    @Test
+    void ObjectWithArrayField() {
+        int[] array = new int[]{1, 0, 0, 1, 1, 1, 0, 0};
+        DocumentWithIntArray object = new DocumentWithIntArray("my-id", array);
```
Comment (roimenashe) [inline src/test/java/org/springframework/data/aerospike/convert/MappingAerospikeConverterTypesTests.java:368]: "my-id" is used a lot in this class, can be extracted to class level field and re-used

**Review by reugn — APPROVED** [review]

**Review by roimenashe — APPROVED** [review]

**Pull request merged**